src/pages/organize/[orgId]/campaigns/[campId]/index.tsx

```diff
@@ -1,11 +1,11 @@
-import { FormattedDate } from 'react-intl';
 import { GetServerSideProps } from 'next';
 import { grey } from '@material-ui/core/colors';
-import { FormattedMessage as Msg } from 'react-intl';
 import NextLink from 'next/link';
-import { Avatar, Box, Button, Link, List, ListItem, makeStyles, Typography } from '@material-ui/core';
+import { Avatar, Box, Button, Link, makeStyles, Typography } from '@material-ui/core';
+import { FormattedDate, FormattedMessage as Msg } from 'react-intl';
 import { Phone, PlaylistAddCheck, Public, Settings } from '@material-ui/icons';
 
+import EventList from '../../../../../components/organize/Event/EventList';
```

Comment:
You have put this component in a directory called "Event". At first glance that looks odd to me. What is the reasoning behind it?